src/generators/AbstractGenerator.ts

```diff
@@ -27,6 +28,7 @@ export interface CommonGeneratorOptions<
   defaultPreset?: P;
   presets?: Presets<P>;
   processorOptions?: ProcessorOptions;
+  inputProcessor?: InputProcessorOptions;
```

Comment:
Please re-use the processorOptions 🙂 Just merge `InputProcessorOptions` into `ProcessorOptions`.